.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,81 @@
+# https://jupyterbook.org/en/stable/publish/gh-pages.html#automatically-host-your-book-with-github-actions
+name: deploy-book
+
+# Only run this when the main branch changes
+on:
+  # Uncomment the 'pull_request' line below to manually re-build Jupyter Book
+  pull_request:
```

Reminder to self: Comment this out before merging.
```suggestion
  # pull_request:
```